test/E2ETest/Microsoft.Test.E2E.AspNet.OData/Build.AspNetCore3x/BulkOperation/EFTests/BulkOperationPatchHandlersEF.cs

```diff
@@ -0,0 +1,410 @@
+﻿//-----------------------------------------------------------------------------
+// <copyright file="BulkOperationPatchHandlersrEF.cs" company=".NET Foundation">
+//      Copyright (c) .NET Foundation and Contributors. All rights reserved. 
+//      See License.txt in the project root for license information.
+// </copyright>
+//------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNet.OData;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.OData.Edm;
+using Microsoft.OData.UriParser;
+
+namespace Microsoft.Test.E2E.AspNet.OData.BulkOperation
+{
+    internal class EmployeeDBContext : DbContext
+    {
+        public EmployeeDBContext()
+        {
+
+        }
+
+        public EmployeeDBContext(DbContextOptions<EmployeeDBContext> options) : base(options)
+        {
+
+        }
+
+        public DbSet<Employee> Employees { get; set; }
+        public DbSet<Friend> Friends { get; set; }
+
+        protected override void OnModelCreating(Microsoft.EntityFrameworkCore.ModelBuilder modelBuilder)
+        {
+            modelBuilder.Entity<Employee>().HasKey(c => c.ID);
+            modelBuilder.Entity<Employee>().Ignore(c => c.SkillSet);
+            modelBuilder.Entity<Employee>().Ignore(c => c.NewFriends);
+            modelBuilder.Entity<Employee>().Ignore(c => c.UnTypedFriends);
+            modelBuilder.Entity<Employee>().Ignore(c => c.InstanceAnnotations);
+            modelBuilder.Entity<Employee>().Ignore(c => c.FavoriteSports);
+
+            modelBuilder.Entity<NewFriend>().Ignore(c => c.InstanceAnnotations);
+            modelBuilder.Entity<UnTypedFriend>().Ignore(c => c.InstanceAnnotations);
+
+            modelBuilder.Entity<Friend>().HasKey(c => c.Id);
+
+            modelBuilder.Entity<NewOrder>().Ignore(c => c.Container);
+            modelBuilder.Entity<MyNewOrder>().Ignore(c => c.Container);
+        }
+    }
+
+    internal class APIHandlerFactoryEF : ODataAPIHandlerFactory
+    {
+        EmployeeDBContext dbContext;
+
+        public APIHandlerFactoryEF(IEdmModel model, EmployeeDBContext dbContext) : base(model)
+        {
+            this.dbContext = dbContext;
+        }
+
+        public override IODataAPIHandler GetHandler(ODataPath odataPath)
+        {
+            if (odataPath != null)
+            {
+                var pathItems = odataPath;
+            }
+
+            return null;
```

Comment:
It doesn't seem like this method is doing anything. What is the purpose of the `pathItems` variable?

Reply:
We are not adding EF tests now